api/fetch.md

```diff
@@ -0,0 +1,118 @@
+# Retrieving and Tranforming Data
+
+The /fetch API provides fast, one-request access to common complex data extraction requirements.
+It allows for fetch submissions in both FlatBuffers and JSON formats, and returns DF4 output format available in both FlatBuffers and JSON encoding.
```

Comment:
is "DF4" a public standard or something we came up with? If the latter, is it defined/explained somewhere that we can link to from here?

Reply:
As far as I know, the only docs on DF4 are in our internal wiki at https://github.com/circonus/circonus-wiki/blob/master/Engineering/DataFormats/DF4.md  We should probably define that somewhere, but can likely do that as a different ticket.  I